Gordon360/Services/ServiceInterfaces.cs

```diff
@@ -12,6 +12,19 @@
 // Namespace with all the Service Interfaces that are to be implemented. I don't think making this interface is required, the services can work fine on their own.
 // However, building the interfaces first does give a general sense of structure to their implementations. A certain cohesiveness :p.
 // </summary>
+namespace Gordon360.Services.RecIM
```

Comment:
You can make this a nested namespace instead, putting ```csharp
namespace RecIM
{
    ...
}
```
inside of the main `Gordon360.Services` namespace below. That will keep them in sync and also improve the organization overall. Please move it to the bottom of the main namespace, so that specialized namespaces are logically organized at the end of the top-level namespace.

Reply:
Got it!